apps/grammy-nestjs/src/decorators/listeners/admin.decorator.ts

```diff
@@ -0,0 +1,8 @@
+import { createBotDecorator } from '../../utils';
+
+/**
+ * Handler for /help command.
+ *
+ * @see https://telegraf.js.org/#/?id=help
```

Comment:
What?

Reply:
that was there :D

Reply:
I see. It shouldn't. Could we remove it in this pull request? The telegraf documentation no longer exists.

Do you know how to find the equivalent links for grammY docs?

Reply:
Thanks!